core/src/com/mygdx/minigolf/model/levels/Course.java

```diff
@@ -0,0 +1,119 @@
+package com.mygdx.minigolf.model.levels;
```

Comment:
Akkurat nå er course et eget course-objekt mens alle andre ting i courset er courseElement-objekter. Kan være en idé å ikke differensiere så mye mellom de to. Er åpen for idéer.